prompterator/utils.py

```diff
@@ -362,3 +380,42 @@ def insert_hidden_html_marker(helper_element_id, target_streamlit_element=None):
 def format_traceback_for_markdown(text):
     text = re.sub(r" ", "&nbsp;", text)
     return re.sub(r"\n", "\n\n", text)
+
+
+def build_function_calling_tooling(json_schema):
+    schema = json.loads(json_schema)
+    function = schema.copy()
+    function_name = function.pop("title")
+    description = (
+        function.pop("description")
+        if function.get("description", None) is not None
+        else function_name
+    )
+    tools = [
+        {
+            "type": "function",
+            "function": {
+                "name": function_name,
+                "description": description,
+                "parameters": function,
+            },
+        }
+    ]
+
+    return tools, function_name
+
+
+def build_response_format(json_schema):
```

Comment:
This function is heavily OpenAI-specific, could you indicate this somehow, e.g. in the function's name?

Reply:
I added `docstring` with description